src/h5web/ConfigProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import type { ReactNode } from 'react';
+import { HeatmapConfigProvider } from './vis-packs/core/heatmap/config';
+import { LineConfigProvider } from './vis-packs/core/line/config';
+
+interface Props {
+  children: ReactNode;
+}
+
+function ConfigProvider(props: Props) {
+  const { children } = props;
+
+  return (
+    <LineConfigProvider>
+      <HeatmapConfigProvider>{children}</HeatmapConfigProvider>
+    </LineConfigProvider>
```

Comment:
It bothers me a bit that we must now add explicitly the `Provider` in this component.

Could we instead keep `ConfigProvider` in the `VIS_DEF` and iterate through it to call the appropriate providers ?

Reply:
Yeah, it did cross my mind but I thought it might be overly complicated. I'll give it a try, see how it looks.

Reply:
It was a fun challenge. I used a `Set` and a `reduce`. Seems to work perfectly!

 ![image](https://user-images.githubusercontent.com/2936402/117110784-f4402700-ad86-11eb-95bb-2026a708ef90.png)

Reply:
:clap: